lib/crawly/engine.ex

```diff
@@ -61,8 +61,15 @@ defmodule Crawly.Engine do
       |> Map.put_new_lazy(:crawl_id, &UUID.uuid1/0)
 
     # Filter all logs related to a given spider
-    if Crawly.Utils.get_settings(:log_to_file, spider_name) do
-      configure_spider_logs(spider_name, opts[:crawl_id])
+    case {Crawly.Utils.get_settings(:log_to_file, spider_name),
+          Code.ensure_loaded?(LoggerFileBackend)} do
+      {true, true} ->
+        configure_spider_logs(spider_name, opts[:crawl_id])
+
+      {true, false} ->
+        Logger.warn(
+          ":logger_file_backend must be installed as a peer dependency if log_to_file config is set to true"
```

Comment:
Can you add in a link to the LoggerFileBackend docs?

Reply:
for better ux.

Reply:
Done